public/locales/et/translation.json

```diff
@@ -112,5 +112,6 @@
     "donate": "annetama",
     "email": "e -kiri",
     "source code": "l\u00e4htekood",
-    "Question": "K\u00fcsitlus"
+    "Question": "K\u00fcsitlus",
+    "question_select": "Vali küsimused"
```

Comment:
Nit, I prefer the translation key to be the actual sentence in english so that its more readable in code.

example: `"Select question set": "Select question set"`
`"Select question set": "Other translation"`